main.py

```diff
@@ -0,0 +1,97 @@
+import math
+
+
+class Coordinate:
+    def __init__(self, x: float, y: float):
+        self.__x = x
+        self.__y = y
```

Comment:
Private fields are not a very common thing to do in Python. It usually makes you write a lot of getters and setters like in Java, which Python developers don't normally do cause they're lazy. Also, private fields are kind of pointless because you can still do something like: ```my_coordinate._Coordinate__x``` and get/set the value.